src/test/kotlin/build/buf/gradle/AbstractBufIntegrationTest.kt

```diff
@@ -72,7 +72,7 @@ abstract class AbstractBufIntegrationTest : IntegrationTest {
                 "-PkotlinVersion=1.7.20",
                 "-PandroidGradleVersion=7.3.0",
             )
-            .withDebug(false) // Enable for interactive debugging
+            .withDebug(true) // Enable for interactive debugging
```

Comment:
Does this affect anything if you don't want to debug? E.g. does it let tests run to completion or does it stop and wait for a debugger to attach?